sdk/ai/azure-ai-projects/azure/ai/projects/aio/operations/_patch.py

```diff
@@ -310,43 +312,50 @@ async def list(
 
 class DiagnosticsOperations(DiagnosticsOperationsGenerated):
 
-    connection_string: Optional[str] = None
-    """ Application Insights connection string. Call `enable()` to populate this property. """
+    _connection_string: Optional[str] = None
+    _get_connection_string_called: bool = False
 
     def __init__(self, *args, **kwargs):
         self._outer_instance = kwargs.pop("outer_instance")
         super().__init__(*args, **kwargs)
 
-    @distributed_trace_async
-    async def enable(self, **kwargs) -> bool:
-        """Enable Application Insights tracing.
-        This method makes service calls to get the properties of the Applications Insights resource
-        connected to the Azure AI Studio Project. If Application Insights was not enabled for this project,
-        this method will return False. Otherwise, it will return True. In this case the Application Insights
-        connection string can be accessed via the `.diagnostics.connection_string` property.
-
-        :return: True if Application Insights tracing was enabled. False otherwise.
-        :rtype: bool
+    async def get_connection_string(self) -> str:
```

Comment:
could there be other connections that app insights? Wondering if the name should be e.g. get_app_insights_connection.

Reply:
I debated this... I originally had it as `get_application_insights_connection_string()` and it seemed too long. Let me get feedback on this and make a change in a separate PR if needed.